app/validation/validator.py

```diff
@@ -193,123 +172,86 @@ def _get_numeric_range_values(self, answer, answer_ranges):
             'max_referred': answer.get('max_value', {}).get('answer_id'),
         }
 
-    def validate_duplicates(self, json_to_validate, ignored_keys, special_key):
-        unique_items = []
-        duplicate_errors = []
+    def validate_duplicates(self, json_to_validate):
+        special_keys = ['id', 'alias']
 
-        for value in self._parse_values(json_to_validate, ignored_keys, special_key):
-            if value in unique_items:
-                duplicate_errors.append(self._error_message('Duplicate {} found. value {}'.format(special_key, value)))
-            else:
-                unique_items.append(value)
+        for special_key in special_keys:
+            unique_items = []
+            duplicate_errors = []
+
+            for value in self._parse_values(json_to_validate, special_key):
+                if value in unique_items:
+                    duplicate_errors.append(self._error_message('Duplicate {} found. value {}'.format(special_key, value)))
+                else:
+                    unique_items.append(value)
 
         return duplicate_errors
```

Comment:
Should be tabbed in once

Reply:
No it shouldn't. otherwise it will never evaluate the 2nd iteration of the loop.
`duplicate_errors = []` should have been outside the `for`